### ISSUE TITLE
508-defect-3 [Chp 31] - Accessibility Issue - Remove extra fieldset from time preferences

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va.gov-team#15483]
## Description
Currently WAVE throws a warning that a fieldset is missing a corresponding legend. However, the field in question already has a paired fieldset and legend, and one additional fieldset which can be removed. 

## Screenshots
![Screen Shot 2020-10-28 at 11.30.37 AM.png](https://images.zenhubusercontent.com/5d68043b330f64311395e92b/b9bf82e1-5221-47dc-926c-ace8ad6ffc1d) 

## Potential Fix
- Set `showFieldLabel` to false to remove the extra fieldset

## Considerations
- Fixing the above issue will result in larger margins between the field label and inputs. This will need to be fixed as well. 

## Acceptance Criteria
- [ ] Accessibility issue has been fixed
- [ ] PR has been submitted

**Comment by micahchiang** [comment]
Moving this to blocked for now, because it should be directly resolved once this PR is merged - https://github.com/department-of-veterans-affairs/vets-website/pull/14785

**Comment by micahchiang** [comment]
system ticket - https://github.com/department-of-veterans-affairs/va.gov-team/issues/15621
